frontend/mono-ui/web/rainmaker/dev-packages/egov-firenoc-dev/src/ui-config/screens/specs/fire-noc/applyResource/nocDetails.js

```
@@ -18,6 +18,22 @@ import {
 import "./index.css";
 import { onchangeOfTenant } from "./propertyLocationDetails";
 
+
+export const getTenantId = () => {
+  const tenantRaw = sessionStorage.getItem('Digit.Employee.tenantId');
+  let tenantValue = null;
+
+  try {
+    const parsed = JSON.parse(tenantRaw);
+    tenantValue = parsed && parsed.value ? parsed.value : null;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)